src/fides/service/privacy_request/privacy_request_service.py

```diff
@@ -496,27 +522,12 @@ def approve_privacy_requests(
         failed: List[BulkUpdateFailed] = []
 
         for request_id in request_ids:
-            privacy_request = self.get_privacy_request(request_id)
-
-            if not privacy_request:
-                failed.append(
-                    BulkUpdateFailed(
-                        message=f"No privacy request found with id '{request_id}'",
-                        data={"privacy_request_id": request_id},
-                    )
-                )
+            result = self._validate_privacy_request_for_bulk_operation(request_id)
```

Comment:
Wouldn't it be better for `_validate_privacy_request...` to raise an exception and return a well-known-object when it returns correctly rather than checking `isinstance`?